src/main/java/com/hypnoticocelot/appengine/helloworldservice/resources/Login.java

```diff
@@ -0,0 +1,14 @@
+package com.hypnoticocelot.appengine.helloworldservice.resources;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Path("/login")
+public class Login {
```

Comment:
Make it LoginResource.

Reply:
Removed the LoginResource for now as it was a dummy file made by me to learn about the project.